pkgs/misc/bootrom-visionfive/default.nix

```diff
@@ -0,0 +1,29 @@
+{ runCommand
+, buildPackages
+, opensbi ? null
+}:
+
+runCommand "bootrom-visionfive" {
```

Comment:
`bootrom` is not the perfect name for this complex as it actually means the on chip readonly first stage bootloader. And in starfive docs this is called `fw_payload.bin.out` or erroneously, `u-boot`. Simply `firmware` can be a better choice.

Reply:
Good point.